src/linelast_solver.cpp

```diff
@@ -417,6 +418,28 @@ void LinElastSolver::SetupDomainBCOperators()
    }
 }
 
+void LinElastSolver::SetParameterizedProblem(ParameterizedProblem *problem)
+{
+   // Set lambda and mu coeffs
+   Vector lambda(numBdr);
+   Vector mu(numBdr);
+
+   function_factory::linelast_disp::fill_lambda(lambda);
+   function_factory::linelast_disp::fill_mu(mu);
```

Comment:
These functions are specific to `linelast_disp` problem. `SetParameterizedProblem` should be generic enough to support other types of problems. In other words, `SetParameterizedProblem` can (and should) be problem-agnostic.

If `fill_lambda` and `fill_mu` functions will be used for any kind of elasticity problems, I suggest changing the name space to be `linelast_problem` that encompassess all elasticity problem. (e.g. `namespace stokes_problem`).

If we expect `lambda` and `mu` to be different problem-by-problem, then we should try to use `problem`'s function pointers instead. Currently `ParameterizedProblem` only has pointers for boundary/right-hand side, but we can try to use them or add more pointers. We can discuss about this implementation.